### ISSUE TITLE
Can't get correct balance value for UAH

### ISSUE THREAD

**Issue by yesworId** [bukson/steampy#295]
Currently, if you are using **get_wallet_balance()** method it will return incorrect balance value. It parses first numbers before the space and return them. It can be fixed by simply adding a space removal to the "**parse_price()**" function.

**Example**: 

User have actual balance value as "15 234,15₴" it returns balance value as "15" instead of "15 234.15".

```python
# FIXED FUNCTION
def parse_price(price: str) -> Decimal:
    price = price.replace(' ', '')  # <- Added space removal
    pattern = '\D?(\\d*)(\\.|,)?(\\d*)'
    tokens = re.search(pattern, price, re.UNICODE)
    decimal_str = tokens.group(1) + '.' + tokens.group(3)
    return Decimal(decimal_str)
```
**Output**:
`15 234.15`

```python
# PREVIOUS ONE
def parse_price(price: str) -> Decimal:
    pattern = '\D?(\\d*)(\\.|,)?(\\d*)'
    tokens = re.search(pattern, price, re.UNICODE)
    decimal_str = tokens.group(1) + '.' + tokens.group(3)
    return Decimal(decimal_str)
```
**Output**:
`15`

**Comment by bukson** [comment]
I will look into pr fixing it, but I need to add more tests

**Comment by bukson** [comment]
Fixed and deployed with 1.1.1
